### PR TITLE
SNOW-1526509 Log connecting area

### PR DESCRIPTION
**Pull request by sfc-gh-knozderko**
### Description
SNOW-1526509 Log connecting area

### Checklist
- [x] Code compiles correctly
- [x] Code is formatted according to [Coding Conventions](../blob/master/CodingConventions.md)
- [x] Created tests which fail without the change (if possible)
- [x] All tests passing (`dotnet test`)
- [x] Extended the README / documentation, if necessary
- [x] Provide JIRA issue id (if possible) or GitHub issue id in PR name

### CODE REVIEW

**Review by sfc-gh-mhofman — APPROVED** [review]
Thanks for that @sfc-gh-knozderko

**Review by sfc-gh-pmotacki — APPROVED** [review]

File: Snowflake.Data/Core/Session/SFSessionProperty.cs [hunk 1]
```diff
@@ -290,6 +291,11 @@ internal static SFSessionProperties ParseConnectionString(string connectionStrin
             return properties;
         }
 
+        internal static string ResolveConnectionAreaMessage(string host) =>
+            host.EndsWith(".snowflakecomputing.cn", StringComparison.InvariantCultureIgnoreCase)
+                ? "Connecting to CHINA Snowflake domain"
+                : "Connecting to GLOBAL Snowflake domain";
```
Comment (sfc-gh-dprzybysz) [inline Snowflake.Data/Core/Session/SFSessionProperty.cs:297]: is Global a good name? we have addresses in Snowflake with `.global.snowflakecomputing.com`

File: Snowflake.Data/Core/Session/SFSessionProperty.cs [hunk 2]
```diff
@@ -290,6 +291,11 @@ internal static SFSessionProperties ParseConnectionString(string connectionStrin
             return properties;
         }
 
+        internal static string ResolveConnectionAreaMessage(string host) =>
+            host.EndsWith(".snowflakecomputing.cn", StringComparison.InvariantCultureIgnoreCase)
```
Comment (sfc-gh-dprzybysz) [inline Snowflake.Data/Core/Session/SFSessionProperty.cs:295]: should we check for com (current one) and all other domains too (as else branch)?

Comment (sfc-gh-mhofman) [reply Snowflake.Data/Core/Session/SFSessionProperty.cs:295]: agree @sfc-gh-dprzybysz, let's have it as here
https://github.com/snowflakedb/snowflake-jdbc/pull/1826

**Review by sfc-gh-mhofman — CHANGES_REQUESTED** [review]

File: Snowflake.Data/Core/Session/SFSessionProperty.cs (hunk 2 above)
Comment (sfc-gh-knozderko) [reply Snowflake.Data/Core/Session/SFSessionProperty.cs]: Done

**Review by sfc-gh-mhofman — APPROVED** [review]

**Pull request merged**